composer.json

```diff
@@ -48,6 +48,7 @@
         "drupal/better_normalizers": "^1.0-beta3",
         "drupal/block_blacklist": "~1.0",
         "drupal/chosen": "~2.4",
+        "drupal/ckeditor_fixed_toolbar": "dev-1.x#1e4a8191ab289b4c91d9a0f2ac6b40e82e294908",
```

Comment:
No stable releases :(
Not much movement in the last 12 months :(

I had a read through the module and it doesn't do much so this dev version is likely ok and it won't be the end of the world if we have to remove it later due to a deprecation or something. 

@boznik Can you have a review of this module (https://git.drupalcode.org/project/ckeditor_fixed_toolbar) and let me know if there is anything concerning in there for you.

Reply:
yeah its really just a dozen lines of code. super slim